avahi-docker.py

```diff
@@ -78,9 +79,16 @@ def register_avahi():
     containers = c.containers()
     for cont in containers:
         info = c.inspect_container(cont['Id'])
-        ip = info['NetworkSettings']['IPAddress']
-        publish(hostname + '_' + info['Name'][1:]+'.local', ip)
-        publish(info['Name'][1:] + '.' + hostname + '.local', ip)
+        hostnames = (
+            hostname + '_' + info['Name'][1:]+'.local',
+            info['Name'][1:] + '.' + hostname + '.local',
+        )
+        ips = [info['NetworkSettings']['IPAddress']]
+        ip6 = info['NetworkSettings']['GlobalIPv6Address']
```

Comment:
I do not use ipv6. When is this setting set, and when is its sibling `LinkLocalIPv6Address` set? Maybe we should also add that one.